src/main/kotlin/com/manimdsl/Compiler.kt

```diff
@@ -93,10 +99,21 @@ class DSLCommandLineArguments : Callable<Int> {
         }
     }
 
+    @Option(names = ["-f", "--open_file_manager_to_generated_files"], description = ["Show the output file in file manager (optional)."])
```

Comment:
Maybe make this just "--open-file", that's a really long thing to type in a command line